src/tests/office_supplies/TestInkToner.java

```diff
@@ -0,0 +1,50 @@
+package office_supplies;
+
+import common.Common;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.WebDriver;
+import pages.office_supplies.InkTonerPage;
+import pages.office_supplies.OfficeSupplies;
+
+/**
+ * Created by ddantas on 4/3/2017.
+ */
+public class TestInkToner {
+
+   protected WebDriver driver;
+
+
+
+    @Before
+    public void setUp() throws Exception {
+        Common cm = new Common();
+
+        driver = cm.getDriver();
+    }
+
+    @Test
+    public void verifyHeaderText() throws Exception {
+
+        InkTonerPage inkTonerPage = new InkTonerPage(driver);
+
+        inkTonerPage = inkTonerPage.navigateToInkTonerpage();
+
+        Assert.assertTrue(inkTonerPage.getHeaderText().getText().contains("Printer Ink Cartridges & Toner Finder"));
```

Comment:
I wanted to check whether all the letters of string  are present in heading. But I realized that I need to use assert Equal method . I will correct my mistake.